pkg/sql/opt/xform/join_funcs.go

```diff
@@ -292,6 +292,7 @@ func (c *CustomFuncs) GenerateLookupJoins(
 //
 // It should be possible to support semi- and anti- joins. Left joins may be
 // possible with additional complexity.
+// TODO(mgartner): update this comment.
```

Comment:
Ack. Though this has nothing to do with lookup joins on virtual tables - this rule is for lookup joins involving virtual computed columns.

Reply:
Addressed here: https://github.com/cockroachdb/cockroach/pull/93042